packages/internal/generated-clients/src/imx-openapi.json

```diff
@@ -11,9 +11,9 @@
       "url": "https://support.immutable.com",
       "email": "support@immutable.com"
     },
-    "version": "3.0.0"
+    "version": "3.0"
```

Comment:
@alexi21 should this be `3.0.0`? The spec was updated as part of the `user-profile` deployment [here](https://github.com/immutable/platform-services/actions/runs/8593182464).

Reply:
yup